File: app/src/main/java/com/example/yura_style/mvp_test/login_MVP/LoginView.java
package com.example.yura_style.mvp_test.login_MVP;

public interface LoginView {
    String getName();

    String getPassword();

    void showNameError(int resId);

    void showPasswordError(int resId);

    void startMainActivity();

    void showLoginIncorrectError(int resId);
}

File: app/src/main/java/com/example/yura_style/mvp_test/login_MVP/LoginView.kt
package com.example.yura_style.mvp_test.login_MVP

interface LoginView {
    val name: String

    val password: String

    fun showNameError(resId: Int)

    fun showPasswordError(resId: Int)

    fun startMainActivity()

    fun showLoginIncorrectError(resId: Int)
}
